net/cert/pki/test_helpers.cc

```diff
@@ -423,7 +423,11 @@ bool ReadVerifyCertChainTestFromFile(const std::string& file_path_ascii,
 std::string ReadTestFileToString(const std::string& file_path_ascii) {
   // Compute the full path, relative to the src/ directory.
   base::FilePath src_root;
+#if defined(STARBOARD)
+  base::PathService::Get(base::DIR_TEST_DATA, &src_root);
```

Comment:
Can we instead modify base/base_paths_starboard.cc to implement support for DIR_SOURCE_ROOT and return the same as DIR_TEST_DATA?

Reply:
Replied here: https://github.com/youtube/cobalt/pull/2616#discussion_r1529054434

Reply:
Actually, this is a good example that I shouldn't make this change. Replacing it with DIR_TEST_DATA breaks cert/internal/crl_unittest.cc. Will change this back.